### ISSUE TITLE
ProudCity Accessibility Dashboard Design

### ISSUE THREAD

**Issue by kevindherman** [proudcity/wp-proudcity#1284]
Dev Tasks:

- [ ] map failures to actual site
- [ ] launch items
- [ ] support@ permissions only

Luke / Kevin

- [ ] Copy edits
- [ ] play with tool
- [ ] Figure out what tasks are useful for actual users

**Comment by aschmoe** [comment]
![screenshot of proudcity dashboard](https://user-images.githubusercontent.com/1634686/36912853-f4799e1c-1dfc-11e8-9ccd-87853e2fc926.jpg)
![screenshot of proudcity dashboard 1](https://user-images.githubusercontent.com/1634686/36912856-f701f99a-1dfc-11e8-981b-4f3c434f5f1e.jpg)
![screenshot of proudcity dashboard 2](https://user-images.githubusercontent.com/1634686/36912861-f968d906-1dfc-11e8-9b79-c463e98f430a.jpg)
![screenshot of proudcity dashboard 3](https://user-images.githubusercontent.com/1634686/36912864-fbe0e6b0-1dfc-11e8-80d8-36bc00f97979.jpg)

**Comment by jlyon** [comment]
Looks awesome @aschmoe

**Comment by aschmoe** [comment]
@lukefretwell @jlyon @kevindherman adding some screenshots of updates to streamline our upcoming meeting

![screenshot of proudcity dashboard 4](https://user-images.githubusercontent.com/1634686/37426406-910b953e-2783-11e8-9eb3-6cd756f617d8.jpg)

- Added a rule passing
- Converted the error / warning / notices to how many rules they are hitting vs the total count

Q's
- [ ] Language (Rules passed, etc)
- [ ] Will it be confusing to see the errors/warnings/notice counts differ from the result page?

![screenshot of proudcity dashboard 5](https://user-images.githubusercontent.com/1634686/37426520-f9a5aefe-2783-11e8-9a03-dd2710bae984.jpg)

- Enabled the naming of the searches, as I think it is a little easier to see whats going on
- View of the type of graph people will have

Q's 
- [ ] If we run the tool every night / per week and largely the graph is mostly just level... is that shitty?
- [ ] Track passing?


![screenshot of proudcity dashboard 6](https://user-images.githubusercontent.com/1634686/37426714-8b678150-2784-11e8-97f8-430caefa9e61.jpg)

- View of the new Passed rule tab

- [ ] What should the text of these be?  Just links to the spec?  The text is currently the same as if it was an error without the link
- [ ] Passed Rules vs Specific Errors?  Should these be converted to display the number of Rules with errors or is the distinction clear / useful?

**Comment by aschmoe** [comment]
- Sub tabs
  - Main 
  - Report
  - Help

- Contextual help on home page
- Contextual help on report page

-[ ] Get Warnings and errors as close to 0 as possible
-[ ] Initially hide notices
-[ ] Name of tool idea 
-[ ] Weekly frequency

**Comment by lukefretwell** [comment]
@aschmoe what do we need by 'Report' here again?

https://github.com/proudcity/wp-proudcity/issues/1284#issuecomment-373534420

**Comment by aschmoe** [comment]
@lukefretwell I believe the idea was to have a couple tabs 

- landing page that provided resources / context, perhaps with a read-out of your last report.
- the report would be the full tool
- help would be some contact flow... seems like could just be link to form

**Comment by lukefretwell** [comment]
@aschmoe can we use check-square or check-circle for the 'Accessibility' tab icon?

https://fontawesome.com/icons?d=gallery&q=check
